types/async-rerun/index.d.ts

```diff
@@ -0,0 +1,13 @@
+
+type callBackFunction = () => Promise<any>;
```

Comment:
```suggestion
type CallBackFunction = () => Promise<any>;
```
it's not enforced, but IMO custom types/interfaces are written with Uppercase (just like TS own types/classes)